pyrovision/models/pyronear_model.py

```diff
@@ -0,0 +1,34 @@
+ # -*- coding: utf-8 -*-
```

Comment:
My bad, I used to add those in the repo, but in Python3, this is no longer required

Reply:
There is this header everywhere in the repo, maybe we can remove this and addapt the header test in an other PR

Reply:
Forgot about the header check, I agree!